sql/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java

```diff
@@ -236,6 +271,138 @@ private Session session() {
         return result;
     }
 
+    private CompletableFuture<Long> executeAlterTableChangeNumberOfShards(AlterTableAnalyzedStatement analysis) {
+        final DocTableInfo table = analysis.table();
+        final boolean isPartitioned = table.isPartitioned();
+        String sourceIndexName;
+        if (isPartitioned) {
+            Optional<PartitionName> partitionName = analysis.partitionName();
+            assert partitionName.isPresent() : "Resizing operations for partitioned tables " +
+                                               "are only supported at partition level";
+            sourceIndexName = partitionName.get().asIndexName();
+        } else {
+            sourceIndexName = table.ident().indexName();
+        }
+        String targetIndexName = sourceIndexName + "_shrinked_" + System.currentTimeMillis();
+        //String targetIndexName = ".resized" + sourceIndexName + String.valueOf(System.currentTimeMillis());
```

Comment:
we should use a name which cannot be used by a user defined table. a `.resized.` prefix would work, a `_shrinked_` suffix not.

Reply:
done